src/main/java/org/ga4gh/drs/model/ContentsObject.java

```diff
@@ -1,5 +1,60 @@
 package org.ga4gh.drs.model;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import org.springframework.lang.NonNull;
+
+import java.net.URI;
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
 public class ContentsObject {
-    
-}
\ No newline at end of file
+    // Required
+    @NonNull
+    private String name;
+
+    // Optional
+    private List<ContentsObject> contents;
+
+    private List<URI> drsUris;
```

Comment:
will the property name `drsUris` cause it to be serialized as `drs_uris`? the property name in the [spec](https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.1.0/docs/#_definitions) appears to be `drs_uri`